integrations/sensu/entity-manager/README.md

```diff
@@ -0,0 +1,120 @@
+## Overview
+
+<!-- Sensu Integration description; supports markdown -->
+
+The Sensu Entity Manager integration provides automated management of agent subscriptions, labels, and annotations. 
+
+The Sensu Entity Manager works with any check or event producer that generates one instruction per line of `event.check.output` in any of the following formats:
+
+* **Subscriptions** 
+
+  One subscription per line.
+
+  ```
+  system/linux
```

Comment:
weird formatting of readme inside all the  block quotes when rendered by the Catalog  ui.  Catalog UI is adding extra indention for the first line.  Catalog UI bug maybe?

Only workaround i can manifest is to add a blank line at the top of the block quote.

Not actionable here.